frontend/src/ui/Breadcrumbs.tsx

```diff
@@ -13,20 +14,24 @@ type Segment = {
     href: string,
 }
 
-export const Breadcrumbs: React.FC<Props> = ({ path }) => (
-    <nav aria-label="breadcrumbs" css={{ marginBottom: 16 }}>
-        <ol css={{ listStyle: "none", padding: 0, margin: 0 }}>
-            <Segment target="/" first active={path.length == 0}>
-                <FontAwesomeIcon title="Home" icon={faHome} />
-            </Segment>
-            { path.map((segment, i) => (
-                <Segment key={i} target={segment.href} active={i == path.length - 1}>
-                    { segment.label }
+export const Breadcrumbs: React.FC<Props> = ({ path }) => {
+    const { t } = useTranslation();
+
+    return (
+        <nav aria-label="breadcrumbs" css={{ marginBottom: 16 }}>
+            <ol css={{ listStyle: "none", padding: 0, margin: 0 }}>
+                <Segment target="/" first active={path.length == 0}>
+                    <FontAwesomeIcon title={t("home")} icon={faHome} />
```

Comment:
Another thing I only notice now, looking at this for a second time: Shouldn't this be

```suggestion
                    <FontAwesomeIcon title={t("home")} icon={ faHome } />
```

Or what are our rules for spacing in `{}` now exactly?

Reply:
Note to self: See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md

We should add that to #41 once there is a decision.

Reply:
If we go with the "what's most popular in the ecosystem" heuristic: `standard`, the Airbnb style guide and the ESLint React plugin's `recommended` rules all prefer `{noSpace}` for all JSX interpolation, not just in attributes.

Reply:
Interesting. Ok, no space in JSX curlies then.